smartsim/_core/_install/builder.py

```diff
@@ -453,17 +480,20 @@ def build_from_git(self, git_url: str, branch: str, device: str = "cpu") -> None
         self.copy_tf_cmake()
 
         # get RedisAI dependencies
-        dep_cmd = [
-            self.binary_path("env"),
-            "WITH_PT=0",  # torch is always 0 because we never use the torch from RAI
-            f"WITH_TF={1 if self.tf and not self.libtf_dir else 0}",
-            "WITH_TFLITE=0",  # never build with TF lite (for now)
-            f"WITH_ORT={self.onnx}",
-            "VERBOSE=1",
-            self.binary_path("bash"),
-            str(self.rai_build_path / "get_deps.sh"),
-            str(device),
-        ]
+        dep_cmd = self._rai_build_env_prefix(
+            with_pt=self.build_torch,
+            with_tf=self.build_tf,
+            with_ort=self.build_onnx,
```

Comment:
very nice pivot from the hardcoded strings!

Reply:
I'd love to say I came up with the idea, but honestly I was just following the advice of a very wise reviewer, hahaha!!